pkg/internal/checkup/checkup.go

```diff
@@ -23,6 +23,7 @@ import (
 	"context"
 	"fmt"
 	"log"
+	"strings"
```

Comment:
Commit message and PR description:
> The checkup may fail on multiple results

s/results/reasons

Reply:
DONE